pip-Makefile.bs4

```diff
@@ -0,0 +1,6 @@
+all:
+	true
+
+install:
+	python3 setup.py install --prefix=/app --root=/ ${ARGS}
+	2to3 -w /app/lib/python3*/site-packages/bs4
```

Comment:
From bs4 source code:
# The Python 2 source is the definitive source. This script uses 2to3-3.2 to
# create a new python3/bs4 source tree that works under Python 3.